cpp/refactor_module/algorithm/refactor.hpp

```diff
@@ -4,6 +4,20 @@
 
 namespace Refactor {
 
+enum class RelSelectStrategy { INCOMING = 0, OUTGOING = 1, MERGE = 2 };
+
+enum class PropertiesStrategy { DISCARD = 0, OVERRIDE = 1, COMBINE = 2 };
+
+struct Config {
+  explicit Config(const mgp::Map &config);
+
+  void SetRelStrategy(std::string_view strategy);
+  void SetPropStrategy(std::string_view strategy);
+
+  RelSelectStrategy rel_strategy;
+  PropertiesStrategy prop_strategy;
```

Comment:
Use _ at the end to make emphasize it is private member, so `rel_strategy_`

Reply:
This is a struct, not a class so ordinary naming convention should apply. I think it should stay a struct because it only carries data